src/app/commands/cmd_change_pixel_format.cpp

```diff
@@ -460,163 +461,82 @@ class ColorModeWindow : public app::gen::ColorMode {
 
 } // anonymous namespace
 
-class ChangePixelFormatCommand : public Command {
+struct ChangePixelFormatParams : public NewParams {
+  Param<bool> ui{ this, false, "ui" };
+  Param<PixelFormat> format{ this, IMAGE_RGB, "format" };
```

Comment:
Change this to `ColorMode`. `PixelFormat` is an deprecated enum to represent the color mode, maybe in the future we could just remove it, or use it as a real "pixel format" (e.g. to use more formats, like R10G10B10A2).

We don't want to publish the `PixelFormat` enum in scripting API, we already have `ColorMode`. 

```c++
  Param<ColorMode> colorMode{ this, ColorMode::RGB, { "colorMode", "format" } };
```

We don't support command aliases yet, only params aliases (like this `"colorMode"/"format"`), but in a future we should change `ChangePixelFormat` to `ChangeColorMode` or something like that.